### PR TITLE
dx-1202 State info regarding partial reversals and captures

### PR DESCRIPTION
**Pull request by arebra**
Reworked "state" in a bunch of response tables so it has info about partial reversals and partial captures.

If someone has a fancy way of making the "partial reversal or capture" in transaction-response.md and transaction-list-response.md change between "partial reversal" or "partial capture" depending on which section the include is used, please make it happen!

### CODE REVIEW

**Review by NullableInt — APPROVED** [review]
I don't know why my review were requested here, but well done 👍

**Comment by NullableInt** [comment]
> 
> 
> Reworked "state" in a bunch of response tables so it has info about partial reversals and partial captures.
> 
> If someone has a fancy way of making the "partial reversal or capture" in transaction-response.md and transaction-list-response.md change between "partial reversal" or "partial capture" depending on which section the include is used, please make it happen!
In the `transaction-list-response.md` you have access to what transaction it is in the `transaction` variable.

**Review by NullableInt — APPROVED** [review]
Yes, this seems to solve your question! Well done.

**Comment by NullableInt** [comment]
Don't forget to `rebase` your branch from the `develop` branch.

**Review by asbjornu — CHANGES_REQUESTED** [review]
Since this is a lot of repeated information, I'd love to see it extracted into a `field-description-state.md` include like the one we have for `field-description-id.md`.

File: _includes/field-description-state.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Indicates the state of the transaction, usually `initialized`, `completed` or
+`failed`. If a partial {{ transaction }} has been done and further transactions
+are possible, the state will be `awaitingActivity`.
```
Comment (asbjornu) [inline _includes/field-description-state.md:3]: I think `_includes/field-description-state.md` needs to take greater care of linefeeds. Checkout out how this is done in `field-description-id.md` for instance.

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/field-description-state.md (hunk 1 above)
Comment (asbjornu) [reply _includes/field-description-state.md:3]: @arebra ☝️

Comment (arebra) [reply _includes/field-description-state.md]: Is this what you meant?

File: _includes/field-description-state.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{{- id_text | strip_newlines -}}
```
Comment (asbjornu) [inline _includes/field-description-state.md:4]:
```suggestion
{%- capture text -%}
Indicates the state of the transaction, usually `initialized`, `completed` or
`failed`. If a partial {{ transaction }} has been done and further transactions
are possible, the state will be `awaitingActivity`.
{%- endcapture -%}
{{- text | strip_newlines -}}
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/field-description-state.md (hunk 1 above)
Comment (asbjornu) [reply _includes/field-description-state.md]: Almost! :)

**Review by asbjornu — APPROVED** [review]

**Pull request merged**